playground/src/Editor/AstralButton.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ButtonHTMLAttributes } from "react";
+import classNames from "classnames";
+
+export default function AstralButton({
+  className,
+  children,
+  ...otherProps
+}: ButtonHTMLAttributes<any>) {
+  return (
+    <button
+      className={classNames(
+        "bg-radiate",
+        "text-black",
+        "hover:text-radiate",
+        "hover:bg-galaxy",
+        "outline-1",
+        "dark:outline",
+        "dark:hover:outline-radiate",
+        "rounded-md",
+        "text-sm",
+        "font-semibold",
+        "enabled:hover:bg-galaxy",
+        className,
+      )}
+      {...otherProps}
+    >
+      {children}
```

Comment:
Maybe we should use small caps for this, like the button on the top right of the homepage: https://astral.sh/. Maybe the same font too? Wdyt?

Reply:
I copied over some styles from astral. Is it okay to commit the fonts?